frontend/src/app/search.tsx

```diff
@@ -98,6 +125,60 @@ export default class Search extends Component {
     );
   }
 
+  private handleChange = (checked, event) => {
```

Comment:
Another good example of specific naming - can we rename this to "handleDeleteCheckboxChange" or similar?

Reply:
Acknowledged.
Yes, this function needs renaming to the task it performs.